tests/migration_test.go

```diff
@@ -1022,6 +1022,90 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				}
 				Expect(timerFrequency).ToNot(BeEmpty())
 			})
+			findTwoNodesWithCloseOrFarTSCFreqencies := func(nodes *k8sv1.NodeList, close bool) (string, string) {
+				for _, node1 := range nodes.Items {
+					freq1, exists := node1.Labels[topology.TSCFrequencyLabel]
+					if !exists {
+						continue
+					}
+					f1, err := strconv.ParseInt(freq1, 10, 64)
+					ExpectWithOffset(1, err).ToNot(HaveOccurred())
+					for _, node2 := range nodes.Items {
```

Comment:
nit: If you want to save some iterations and want to cover the case two nodes have the same frequency
```suggestion
				if len(nodes.Items) == 0 {
				        return "", ""
				}
				for _, node1 := range nodes.Items[:len(nodes.Items) - 1] {
					freq1, exists := node1.Labels[topology.TSCFrequencyLabel]
					if !exists {
						continue
					}
					f1, err := strconv.ParseInt(freq1, 10, 64)
					ExpectWithOffset(1, err).ToNot(HaveOccurred())
					for _, node2 := range nodes.Items[i+1:] {
```